src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/anonymization/SpecialAnonymizationUserPage.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.HelpPage;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.net.URLEncoder;
+
+public class SpecialAnonymizationUserPage extends SpecialPageObject {
+
+  @FindBy(css = "#username")
+  private WebElement AnonymizationTextBox;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "AnonymizationTextBox" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)